CHORE: Excluded IDE directories and added yarn.lock file.

bouncydragon opened the pull request:
- I excluded the IDE directories, we can standardize on how we code using the `Eslint` and `Prettier`.
- I added `yarn.lock` for users who use `yarn`.

davidrayc commented:
> * I excluded the IDE directories, we can standardize on how we code using the `Eslint` and `Prettier`.

How do you suggest we standardize eslint and prettier?

> * I added `yarn.lock` for users who use `yarn`.

We cannot have 2 pkg managers. Why can't you use pnpm?

bouncydragon commented:
> How do you suggest we standardize eslint and prettier?
- There are of rules in both of them which can we use to standardize the coding experience. 
e.g Prettier: `bracketSpacing`, `trailingComma`, `printWidth` ... .etc.
For Eslint you can refer to the next.js docs: https://nextjs.org/docs/app/building-your-application/configuring/eslint



> We cannot have 2 pkg managers. Why can't you use pnpm?
- I haven't really used `pnpm`, I used `yarn` all the time and we haven't establish what package manager we'll be using (correct me if I'm wrong, I may be out during the meeting) that's why I added the `yarn.lock`. If we'll go with `pnpm` I have no complaints.

EDIT: I've updated the PR and removed the `yarn.lock`

davidrayc commented:
> There are of rules in both of them which can we use to standardize the coding experience.
> e.g Prettier: bracketSpacing, trailingComma, printWidth ... .etc.
> For Eslint you can refer to the next.js docs: https://nextjs.org/docs/app/building-your-application/configuring/eslint

Ok. We set those rules in prettierrc and eslintrc.json which we already have in this workspace. How is this related to removing .vscode and other IDE-specific folders?
